_posts/en/newsletters/2020-05-27-newsletter.md

```diff
@@ -0,0 +1,178 @@
+  waiting a slightly different amount of time for each peer (or group of
+  peers) before announcing new transactions to them, causing each
+  transaction to propagate across the network using a different path.
+  Randomizing the propagation path of each transaction makes it less
+  reliable for surveillance nodes to assume that the first node announcing
+  a transaction was the first node to receive it.
+
+- [Bitcoin Core #17681][] allows the wallet to internally derive new
+  addresses for a [BIP32][] HD wallet seed even after that seed is no
+  longer the wallet's active seed.  This makes it safe to switch to a
+  new HD seed with the `sethdseed` (set HD seed) RPC even while the node
+  is performing an initial block chain download, such as when restoring
+  a wallet backup on a newly-started node---the updated code ensures the
+  wallet will see any payments to addresses previously derived from the
+  old HD seed.
+
+- [Bitcoin Core #18895][] updates the RPCs that return data about
+  individual transactions in the mempool (e.g. `getrawmempool` and
+  `getmempoolentry`) with an `unbroadcast` field that indicates whether
+  or not any of the local node's peers have requested a copy of the
+  transaction (see [Newsletter #96][news96 unbroadcast] for a summary of
+  broadcast tracking).  Additionally, the `getmempoolinfo` RPC is
+  updated with an `unbroadcastcount` field indicating the number of
+  unbroadcast transactions.  For privacy, the broadcast status of a
+  transaction is only tracked if it was submitted by either the node's
+  wallet or the `sendrawtransaction` RPC.
+
+- [Bitcoin Core #18677][] adds a new `--enable-multiprocess` build
+  configuration option that will produce additional binaries alongside
+  the existing `bitcoind` and `bitcoin-qt` binaries.  For now, the only
+  difference between the new and old binaries is their name.  However,
+  if [PR #10102][Bitcoin Core #10102] is merged, the new binaries will
+  split the functions of node, wallet, and GUI into separate executables
+  that communicate with each other when necessary.  The build option is
+  currently disabled by default.  See also [Newsletter #39][news39
+  multiprocess] for the last time we wrote about the multiprocess
+  sub-project.
+
+- [Bitcoin Core #18594][] allows the `bitcoin-cli` command to print the
+  balances of each wallet loaded in multiwallet mode using the
+  `-getinfo` command line parameter.
+
+- [C-Lightning #3738][] adds initial support for [BIP174][] Partially
+  Signed Bitcoin Transactions ([PSBT][topic psbt]), making use of
+  [libwally's PSBT support][libwally psbt].  The only user-visible change
+  is that the PSBT form of the transaction is returned by the
+  `txprepare` RPC, but the PR is tagged in GitHub as working towards
+  dual funding of new channels (see [Newsletter #83][news83 interactive]
+  about using PSBT for interactive construction of funding transactions).
```

Comment:
```suggestion
 using PSBT for interactive construction of funding transactions).
```

Reply:
Skipping this suggestion so that I can make a different change in the doc, but thanks!